rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/identify/CompoundIdentifier.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.windup.rules.apps.java.archives.identify;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.windup.rules.apps.java.archives.identify.api.ArchiveHashIdentifier;
+
+/**
+ *
+ *  @author Ondrej Zizka, ozizka at redhat.com
+ */
+public class CompoundIdentifier implements ArchiveHashIdentifier
```

Comment:
I do wish that there were a better name than this. Identifier is a noun, but it appears to be used as a verb in this case. Perhaps something like "HashToIdentifierMapping" (just an idea... I don't love this name either)?